ARCdigi/src/ARCdigitizer.cpp

```diff
@@ -0,0 +1,87 @@
+#include "ARCdigitizer.h"
+
+// EDM4HEP
+#include "edm4hep/Vector3d.h"
+
+// DD4HEP
+#include "DDRec/CellIDPositionConverter.h"
+
+// ROOT
+#include "TRandom.h"
+
+// STL
+#include <unordered_map>
+#include <utility>
+
+DECLARE_COMPONENT(ARCdigitizer)
+
+ARCdigitizer::ARCdigitizer(const std::string& aName, ISvcLocator* aSvcLoc) : GaudiAlgorithm(aName, aSvcLoc) {
+  declareProperty("inputSimHits", m_input_sim_hits, "Input sim tracker hit collection name");
+  declareProperty("outputDigiHits", m_output_digi_hits, "Output digitized tracker hit collection name");
+  declareProperty("detectorCompact", m_det_compact, "Path to detector compact");
```

Comment:
To be removed when retrieving geometry from the already built instance

Reply:
Done!